frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/vm-settings-tab.tsx

```diff
@@ -88,6 +88,7 @@ export class VMSettingsTabComponent extends React.Component<VMSettingsTabCompone
             key={VMSettingsField.USER_TEMPLATE}
             userTemplateField={this.getField(VMSettingsField.USER_TEMPLATE)}
             userTemplates={userTemplates}
+            fieldValue={this.getFieldValue(VMSettingsField.USER_TEMPLATE)}
```

Comment:
this is already present in `userTemplateField`

Reply:
done